tests/coverage-map/status-quo/issue-84561.cov-map

```diff
@@ -1,11 +1,10 @@
 Function name: <issue_84561::Foo as core::cmp::PartialEq>::eq
-Raw bytes (14): 0x[01, 01, 00, 02, 01, 04, 0a, 00, 13, 00, 00, 0a, 00, 13]
+Raw bytes (9): 0x[01, 01, 00, 01, 01, 04, 0a, 00, 13]
```

Comment:
@Zalathar This commit blesses the coverage map. However, I have no idea how to check that it is still correct. What do you recommend?

Reply:
As a rule of thumb, any PR that doesn't touch coverage-specific code should feel free to just re-bless these.

(They're unfortunately quite sensitive to changes in MIR lowering/optimization, in a way that tends to result in different but equivalent mappings.)

Reply:
In particular, as long as the `run-coverage` tests still pass, then any changes to the mappings are probably just noise.

(Those tests only run if the profiler runtime is enabled in `config.toml`, so they don't run by default in dev builds or on PRs, but they do run in the full CI before merging.)

Reply:
Looking at the actual diff, it seems like some MIR statements that were previously being optimized out are no longer being optimized out. Nothing to worry about from a coverage perspective.